cluster/docker-cluster/provision/verify.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Verify if k8s cluster works
+function verify() {
```

Comment:
We have a ```kubectl``` method that verifies health of the cluster. It would require installing and configuring ```kubectl``` properly so I'm not insisting on doing it in this PR, but I think we should at least leave a TODO for this.